src/uct/tcp/tcp_iface.c

```diff
@@ -542,53 +542,6 @@ static ucs_status_t uct_tcp_iface_listener_init(uct_tcp_iface_t *iface)
     return status;
 }
 
-static ucs_status_t uct_tcp_iface_set_port_range(uct_tcp_iface_t *iface,
```

Comment:
`ucs_sockaddr_get_ip_local_port_range` is no longer used.
shall we remove it? or probably we can leave for future reuse somewhere?